packs/data/abomination-vaults-bestiary.db/dread-wisp.json

```diff
@@ -300,6 +303,14 @@
                             "dim": 20,
                             "shadows": 0.1
                         }
+                    },
+                    {
+                        "domain": "all",
+                        "key": "RollOption",
+                        "label": "Dread Flickering",
```

Comment:
```suggestion
```
It will auto generate a label with the ability name, so skipping making one manually means translators won't have to mess with the REs on the ability.